framework/tests/value/ByteArrayWithNonLiteralConstants.java

```diff
@@ -0,0 +1,11 @@
+import org.checkerframework.common.value.qual.IntVal;
+import org.checkerframework.common.value.qual.StringVal;
+
+public class ByteArrayWithNonLiteralConstants {
```

Comment:
Let's also change the name of this test to something to do with char arrays instead of byte arrays.